src/test/fuzz/block.cpp

```diff
@@ -35,15 +35,19 @@ void test_one_input(const std::vector<uint8_t>& buffer)
     } catch (const std::ios_base::failure&) {
         return;
     }
-
     const Consensus::Params& consensus_params = Params().GetConsensus();
-    CValidationState validation_state_pow_and_merkle;
+
+    BlockValidationState validation_state_contextual;
+    const CBlockIndex* pindex_prev = nullptr;
+    (void)ContextualCheckBlock(block, validation_state_contextual, consensus_params, pindex_prev);
```

Comment:
in commit c5d08da5c38e4140f4b9465f3e8159bcd7874ab2:

On a second thought I am not sure if it makes sense to expose this internal function for the sole purpose to pass a nullptr to it, which makes it execute pretty much no code paths anyway. Also, passing a nullptr never happens in reality.

Reply:
OK, reverted to previous version (see https://github.com/bitcoin/bitcoin/pull/17071#discussion_r356756648).

Please re-review :)

Reply:
I think there is a trade-off of unit and fuzz testing of internal functions that are not exposed in the header. In this particular instance `pindex_prev` is passed as a nullptr to `ContextualCheckBlock`, which then runs almost no code  paths anyway. Not sure what the potential gains are of testing something that can never happen in reality.

Reply:
OK, dropped called to `ContextualCheckBlock`. Please re-review :)